### ISSUE TITLE
Tooltips do not dismiss on ctrl

### ISSUE THREAD

**Issue by Rainhale** [microsoft/fluentui#14987]
We should dismiss via ctrl key as escape closes many dialogs, this is the current behavior in Office.com and was discussed with SME in Control Club and would be great to incorporate into Fluent. Since the esc key propagates while the tool tip is dismissed controls using esc still see normal esc behaviors, on key down of ctrl is a more robust solution.

text bellow is largely copied and is a modified version from [this issue](https://github.com/microsoft/fluentui/issues/9435):
https://codepen.io/dbjorge/pen/QXbvzN 

Actual behavior:
Go to the codepen
Click the "Hover Over Me" element
Press "Tab" then "Shift+Tab" to simulate keyboard navigating to element
Wait for tooltip to appear
Press left or right "ctrl"
Observe that tooltip does not disappear
Expected behavior:
tooltip should be dismissable using ctrl key

Documentation describing expected behavior
From https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

Violation of Success Criterion 1.4.13 Content on Hover or Focus

**Comment by paulgildea** [comment]
@Rainhale Thanks for the feedback.

@smhigley It would be helpful for you to weigh in on this desired behavior. 

I'm not opposed to adding if that's the current requirement.
